### PR TITLE
Use Grid layout for Conference listing

### PR DESCRIPTION
**Pull request by mannar-1**
image
modified cards layout to above

### CODE REVIEW

File: junction/templates/pages/home_page.html [hunk 1]
```diff
@@ -7,6 +7,11 @@
 
 {% block page_classes %}{{ block.super }} homepage{% endblock %}
 {% block breadcrumbs %}{% endblock %}
+
+{% block extra_css %}
+<link rel="stylesheet" href="{% static 'css/homepage.css' %}">
```
Comment (ananyo2012) [inline junction/templates/pages/home_page.html:12]: This isn't used anywhere. Can be removed.

File: junction/static/css/app.css [hunk 2]
```diff
@@ -8993,3 +8993,97 @@ textarea.review-form select {
...
+  }
+}
+
+@media (max-width: 768px) {
+  #conference-grid {
+      grid-template-columns: 1fr;
+  }
+}
+
+html {
```
Comment (RajatRajdeep) [inline junction/static/css/app.css:9079]: This PR is specifically for the Conference List. Please remove the changes in the html tag as it applies to the entire app.

File: junction/static/css/app.css [hunk 3]
```diff
@@ -8993,3 +8993,97 @@ textarea.review-form select {
...
+}
+
+.conference-item:hover {
+  box-shadow: 0 5px 15px rgba(0,0,0,0.2);
+  transform: translateY(-5px);
+  text-decoration: none;
+}
+
+.conference-date {
+  font-size: 0.9em;
```
Comment (RajatRajdeep) [inline junction/static/css/app.css:9043]: Please use px for consistency, as it is already being used in all other UI styles.

**Review by RajatRajdeep — CHANGES_REQUESTED** [review]
Could you provide a screencast demonstrating the changes, transitions, and functionality across different screen sizes?

File: junction/templates/pages/home_page.html (hunk 1 above)
Comment (mannar-1) [reply junction/templates/pages/home_page.html:12]: ok

**Comment by mannar-1** [comment]
mobile:
image
Tab:
image
Laptop:
image

File: junction/static/css/app.css (hunk 2 above)
Comment (mannar-1) [reply junction/static/css/app.css]: removed it .

File: junction/templates/pages/home_page.html (hunk 1 above)
Comment (mannar-1) [reply junction/templates/pages/home_page.html]: removed please check

**Comment by mannar-1** [comment]
I made required changes and pushed what changes you told. please check ones

File: junction/templates/pages/home_page.html [hunk 4]
```diff
@@ -7,6 +7,8 @@
 
 {% block page_classes %}{{ block.super }} homepage{% endblock %}
 {% block breadcrumbs %}{% endblock %}
+
```
Comment (ananyo2012) [inline junction/templates/pages/home_page.html:10]: Please remove blank lines

**Review by RajatRajdeep — APPROVED** [review]

**Comment by ananyo2012** [comment]
Thanks @mannar-1 . A good step towards UI re-design.

**Pull request merged**